common/locales/en/groups.json

```diff
@@ -39,6 +39,8 @@
     "editGroup": "Edit Group",
     "newGroupName": "<%= groupType %> Name",
     "groupName": "Group Name",
+    "groupLeader": "Group Leader: ",
+    "groupID": "Group ID: ",
```

Comment:
"Group leader" wasn't needed before this PR, but I was really surprised that "Group ID" wasn't already a string.